### ISSUE TITLE
[BUG] keka no longer serves as default app

### ISSUE THREAD

**Issue by MRKsas** [aonez/Keka#1197]
### Configuration
- Keka version: Version 1.2.58
- macOS version: 13.1 (22C65)

### Describe the bug
while keka is the default app, mac's built-in app Archive Utility take over keka's job.

https://user-images.githubusercontent.com/115054265/207817349-b5a73fba-49d8-4a22-99b7-6420c414cc89.mp4

**Comment by aonez** [comment]
Can you show what appears on the same file while right click -> "Open with". Also please create a new file with Keka and see if it reproduces.

I'm unable to reproduce the issue at this point.

**Comment by MRKsas** [comment]
https://user-images.githubusercontent.com/115054265/207849100-e9b51c63-79f4-4dff-8658-7438bd71f617.mp4

**Comment by MRKsas** [comment]
the issue appears after I upgrade mac os to 13.1, which might be an explanation.

**Comment by aonez** [comment]
Strange, I’ve tested it on the latest version of Ventura.

Did you used the Keka - Preferences - “Set Keka as default uncompressor”? What happens if you do it again?

**Comment by Manan2004** [comment]
Same for me But it changed to The Unarchiver instead of Mac default

It does show keka when
right click - Open with - Keka (default)

**Comment by aonez** [comment]
@Manan2004 setting Keka as default again using the Preferences option “Set Keka as default uncompressor” doesn’t fix it for you neither?

You’ve also just updated to Ventura 13.1?

Probably the workaround @MRKsas did, assign another app as the default, click on “change all” and set Keka again.

**Comment by Manan2004** [comment]
Yes and yes and yes the workaround worked i just did right click - get info - keka was selected already but i just selected it again and change all that fixed it

**Comment by tr1s** [comment]
The workaround worked for me last time and then reset... I'm not exactly sure what/when it reset but it did.

I've since upgraded to Vantura 13.3 and have done the workaround again and it's working. I'll keep an eye on if/when it resets to the built-in extractor again.

@aonez to answer your question from #1225, no I don't use any other third-party apps and I don't have any other users to test.

**Comment by aonez** [comment]
I've now have this issue too for ZIP files, even if Keka is set as the default app. In my test **restarting the Finder** solved the issue.

At this point I just hope macOS 14 is better and less buggy than Ventura has been.

**Comment by Zephyr-8** [comment]
Same problem here and same workaround fixed. 

version: 15.6.1
